### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by colinodell**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | n/a |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

Github has support for issue and template files (`issue_template.md` and `pull_request_template.md`).  The contents of these files will auto-populate new issues/PRs.

This PR creates a `PULL_REQUEST_TEMPLATE.md` file using the [Symfony Pull Request Template](https://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request).

See https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

**Comment by javiereguiluz** [comment]
:+1: great!

**Comment by dosten** [comment]
Do you know how this works? If the `PULL_REQUEST_TEMPLATE.md` is present in 2.7+ but not in 2.3 and I open a PR in 2.3 the PR body will be empty?

**Comment by colinodell** [comment]
Github has made an official post on this feature: https://github.com/blog/2111-issue-and-pull-request-templates

**Comment by stloyd** [comment]
IMO this should land as suggested by GitHub team in the `.github/` folder (also with other GH related files).

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | [yes|no]
+| New feature?  | [yes|no]
+| BC breaks?    | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass?   | [yes|no]
+| Fixed tickets | [comma separated list of tickets fixed by the PR]
+| License       | MIT
+| Doc PR        | [The reference to the documentation PR if any]
```
Comment (Tobion) [inline PULL_REQUEST_TEMPLATE.md]: "The" can be removed

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | [yes|no]
+| New feature?  | [yes|no]
+| BC breaks?    | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass?   | [yes|no]
+| Fixed tickets | [comma separated list of tickets fixed by the PR]
```
Comment (Tobion) [inline PULL_REQUEST_TEMPLATE.md]: maybe we should add "if any" here as well. I've seen alot of people creating issues just to reference them in the PR. Or they self-reference the PR itself.

**Comment by colinodell** [comment]
@Tobion I've made those two changes.

**Comment by Tobion** [comment]
IMO the template should be removed from https://github.com/symfony/symfony/blob/master/CONTRIBUTING.md then

**Comment by WouterJ** [comment]
:+1: for moving the file to `.github/`, as mentioned by the official blog post.

**Comment by colinodell** [comment]
Alright I've moved the file to `.github/` and removed the template from CONTRIBUTING.  Let me know if you'd like to see any other changes.

**Comment by Taluu** [comment]
You should also move the `CONTRIBUTING` inside the `.github` directory

**Comment by colinodell** [comment]
@Taluu While GitHub does support that, I personally like seeing the `CONTRIBUTING` file in the root.  I tend to check that first instead of relying on GitHub to display the notice when creating the issue/PR later on.  Just my $0.02.

**Comment by fabpot** [comment]
Thank you @colinodell.

**Pull request closed without merging**